src/Microsoft.AspNetCore.Mvc.Core/Internal/ControllerActionInvoker.cs

```diff
@@ -657,6 +675,9 @@ private Task Next(ref State next, ref Scope scope, ref object state, ref bool is
                             return task;
                         }
 
+                        // Found this while investigating #5594. This is not right, but we think it's harmless
+                        // so we're leaving it here for the patch release. This should go to InvokeEnd if the
+                        // scope is not Resource because that can only happen when there are no resource filters.
```

Comment:
And, of course, my requested change is to double the number of comments like this.